specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2023-07-01/imagebuilder.json

```diff
@@ -0,0 +1,2082 @@
+        "source",
+        "distribute"
+      ]
+    },
+    "ImageTemplateIdentity": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of identity used for the image template. The type 'None' will remove any identities from the image template.",
+          "enum": [
+            "UserAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceIdentityType",
+            "modelAsString": false
+          }
+        },
+        "userAssignedIdentities": {
+          "$ref": "#/definitions/UserAssignedIdentities"
+        }
+      },
+      "description": "Identity for the image template."
+    },
+    "UserAssignedIdentities": {
+      "title": "User-Assigned Identities",
+      "description": "The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.",
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/UserAssignedIdentity"
+      }
+    },
+    "UserAssignedIdentity": {
+      "type": "object",
+      "description": "User assigned identity properties",
+      "properties": {
+        "principalId": {
+          "description": "The principal ID of the assigned identity.",
+          "type": "string",
+          "readOnly": true
+        },
+        "clientId": {
+          "description": "The client ID of the assigned identity.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "RunOutputProperties": {
+      "type": "object",
+      "description": "Describes the properties of a run output",
+      "properties": {
+        "artifactId": {
+          "type": "string",
+          "description": "The resource id of the artifact."
+        },
+        "artifactUri": {
+          "type": "string",
+          "description": "The location URI of the artifact."
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "Provisioning state of the resource",
+          "readOnly": true
+        }
+      }
+    },
+    "ImageTemplate": {
+      "type": "object",
+      "description": "Image template is an ARM resource managed by Microsoft.VirtualMachineImages provider",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ImageTemplateProperties",
+          "description": "The properties of the image template"
+        },
+        "identity": {
+          "$ref": "#/definitions/ImageTemplateIdentity",
+          "description": "The identity of the image template, if configured."
+        },
+        "distribute": {
```

Comment:
I believe `distribute` should be part of the `properties` bag?

Reply:
It is inside of properties bag in line 1696. Do you mean it needs to be inside of the 2nd properties bag in line 1697?

Reply:
Since the `distribute` is already added in the definition of "ImageTemplateProperties", there's no need to define the `distribute` here. And it's not allowed to define it here. Please remove the `distribute` definition at line 1703.

Reply:
okay removed at line 1703

Reply:
i removed distribute but there is still 1 LintDiff error. how can i fix?